src/components/User/User.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './User.scss';
+
+export const User = ({ user }) => {
+  const { name } = user;
+
+  return (
+    <h1 className="name">{name}</h1>
```

Comment:
page can only have ONE h1 element

```suggestion
    <h3 className="name">{name}</h3>
```